build.sbt

```diff
@@ -151,6 +151,56 @@ lazy val noPublishSettings = Seq(
   publishArtifact := false
 )
 
+// https://github.com/lightbend/mima/issues/388
+lazy val mimaSettings = Def.settings(
```

Comment:
This is not pretty but I guess we will be able to remove this once that issue is fixed